eng/pipelines/scripts/mgmt_sdk_compiler.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+#args expected
+# $1. Java version : 1.7 or 1,8
+
+echo "#### CWD : "
+pwd
+
+echo "#### versions of java available:" 
+ls /usr/lib/jvm
+
+JAVA7HOME="/usr/lib/jvm/zulu-7-azure-amd64"
```

Comment:
hard-coding these is not a great idea. Why not use the Java Tool Installer task (https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/tool/java-tool-installer?view=azure-devops)?

Reply:
As we chatted if yout his is working now I'm fine with it but please file a follow-up issue to investigate using the Java Tool Installer instead of us needing to maintain the logic for selecting the correct java install.